components/admin_console/brand_image_setting/brand_image_setting.jsx

```diff
@@ -17,6 +17,11 @@ const HTTP_STATUS_OK = 200;
 export default class BrandImageSetting extends React.PureComponent {
     static propTypes = {
 
+        /*
+         * Set for testing purpose
+         */
+        id: PropTypes.string,
```

Comment:
@nadalfederer, can you elaborate on where this `id` is being set right now?

Reply:
No, I have not been able to find it. I just checked that it has this property for testing